WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowCompletionViewController.swift

```diff
@@ -129,6 +134,16 @@ class BloggingRemindersFlowCompletionViewController: UIViewController {
         calculatePreferredContentSize()
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if traitCollection.preferredContentSizeCategory.isAccessibilityCategory {
```

Comment:
Nitpick, we could do `hintLabel.isHidden = traitCollection.preferredContentSizeCategory.isAccessibilityCategory`